### PR TITLE
Review Widget

### PR DESCRIPTION
**Pull request by dsilberger**

### CODE REVIEW

File: client/src/components/reviews/ProductBreakdown.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import ProductBreakdown from './ProductBreakdown.js';
+
+it('renders learn react link', () => {
```
Comment (chengmuy) [inline client/src/components/reviews/ProductBreakdown.test.js]: Can you update this test name?

File: client/src/components/reviews/RatingsBreakdown.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import RatingsBreakdown from './RatingsBreakdown.js';
+
+it('renders learn react link', () => {
```
Comment (chengmuy) [inline client/src/components/reviews/RatingsBreakdown.test.js]: Can you update this test name?

File: client/src/components/reviews/ReviewList.test.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import ReviewList from './ReviewList.js';
+
+it('renders learn react link', () => {
```
Comment (chengmuy) [inline client/src/components/reviews/ReviewList.test.js]: Can you update this test name?

File: client/src/components/reviews/ReviewTile.test.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import ReviewTile from './ReviewTile.js';
+
+it('renders learn react link', () => {
```
Comment (chengmuy) [inline client/src/components/reviews/ReviewTile.test.js]: Can you update this test name?

File: client/src/components/reviews/index.test.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import ReviewWidget from './index';
+
+it('renders learn react link', () => {
```
Comment (chengmuy) [inline client/src/components/reviews/index.test.js]: Can you update this test name?

Comment (dsilberger) [reply client/src/components/reviews/index.test.js]: Thanks for the catch. Tests now accurately describe themselves.

**Pull request merged**